dts/bindings/interrupt-controller/intel,ioapic.yaml

```diff
@@ -0,0 +1,33 @@
+---
+title: Intel I/O Advanced Programmable Interrupt Controller
+version: 0.1
+
+description: >
+    This binding describes the Intel I/O Advanced Programmable Interrupt
+    controller
+
+properties:
+  - compatible:
+      category: required
+      type: string
+      description: compatible strings
+      constraint: "intel,ioapic"
+
+  - reg:
+      category: required
+      type: int
+      description: mmio register space
+      generation: define
+
+  - arm,num-irq-priority-bits:
+      category: required
+      type: int
+      description: number of bits of IRQ priorities
+      generation: define
+
+cell_string: IRQ
+
+"#cells":
+  - irq
+  - priority
```

Comment:
it doesn't appear that the ioapic supports priority.  The second cell should be interrupt "sense":

  Device's interrupt property:

     interrupts = <P S>;

  The first number (P) represents the interrupt pin which is wired to the
  IO APIC. The second number (S) represents the sense of interrupt which
  should be configured and can be one of:
    1 - Edge Rising/High
    2 - Edge Falling/Low
    4 - Level High
    8 - Level Low

We should introduce a include/dt-bindings/interrupt-controller/irq.h with these defines.

Reply:
ok.